src/figuresClasses.ts

```diff
@@ -1,19 +1,98 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
it will be great to create `enum` typings for shape and color and use it everywhere